earth_enterprise/tutorial/FusionTutorial/download_tutorial.sh

```diff
@@ -3,6 +3,11 @@
 TMPFILE="/tmp/FusionTutorial-Full.tar.gz"
 SCRIPTDIR=`dirname "$0"`
 
+if [ "$EUID" != "0" ] ; then
```

Comment:
Minor: I think this only works in BASH, and not in other POSIX shell implementations, like `dash` on Ubuntu. You could use `id -u` to be more portable.